endoreg_db/models/media/video/video_file.py

```
@@ -768,64 +778,40 @@ def create_video_without_outside_frames(
+            extracted = video.extract_frames(quality=2, overwrite=False, ext="jpg", verbose=False, from_processed=True)
             assert extracted is True
         except AssertionError:
-            extracted = video.extract_frames(
-                quality=2,
-                overwrite=False,
-                ext="jpg",
-                verbose=False,
-                from_processed=True,
-            )
+            # Use default anonymization here
+            video.anonymize
```

Ruff (0.14.6)

Found useless expression. Either assign it to a variable or remove it.

(B018)